governance/scripts/deploy_multisig_mods

```diff
@@ -6,20 +6,53 @@
  *
  * Usage: scripts/deploy_multisig_mods
  *
+ * ## How to deploy Zodiac modules
+ *
+ * 1. get the modules code
+ * git clone git@github.com:gnosisguild/zodiac-modifier-delay.git
+ * git clone git@github.com:gnosisguild/zodiac-module-connext.git
+ *
+ * 2. prepare repo
+ * yarn
+ * yarn build
+ *
+ * 3. add required network in `hardhat.config.ts` with the following pattern
+ * `
+ * base: {
+ *     url: "https://rpc.unlock-protocol.com/8453",
+ *     accounts: [process.env.DEPLOYER_PRIVATE_KEY],
+ *   },
+ * `
+ * and block explorer api key
+ *
+ * `
+ * etherscan: {
+ *    apiKey: {
+ *      base: '<your-api-key>',
+ *    }
+ * },
+ * `
+ *
+ * 4. export DEPLOYER_PRIVATE_KEY to env
+ * 5. run this script to get the commands
+ *
+ * NB: the first run will give you instructions for delay module. The second
+ * one for Connext module.
+ *
  */
 
 const allNetworks = require('@unlock-protocol/networks')
 
 const owner = '0x246A13358Fb27523642D86367a51C2aEB137Ac6C'
-const daoTimelockAddress = '0x17eedfb0a6e6e06e95b3a1f928dc4024240bc76b'
+const daoTimelockAddress = '0x17EEDFb0a6E6e06E95B3A1F928dc4024240BC76B'
 const daoChainId = 1
 const daoDomainId = allNetworks.networks[daoChainId].governanceBridge.domainId
 
 // in seconds
 const ONE_DAY = 24 * 3600
 
 // if any network is present this array, only these will be executed
-const only = ['polygon', 'gnosis', 'bsc', 'arbitrum', 'optimism']
+const only = ['polygon', 'gnosis', 'bsc', 'arbitrum', 'optimism', 'base']
```

Comment:
We should mobve that config to the package network. Maybe a flag there?

Reply:
I dont think we gain much by automating it here. The script is just a parser to show command line, it does not actually execute anything